client/src/App.jsx

```diff
@@ -33,20 +33,37 @@ export default class App extends Component {
   };
 
   componentDidMount() {
-    socket.emit('getmails');
+    socket.on('error', data => console.log(data));
+    socket.on('request getmails', ()=> { 
+      socket.emit('getmails', {range:'1:5'});
+    })
     socket.on('mails', data => {
-      this.setState({
-        tickets: [...this.state.tickets, JSON.parse(data)],
-      });
+      console.log(data)
     });
-    socket.on('notification', () => {
-      console.log('notification');
-      socket.emit('get new mail')
-    });
-    socket.on('new mail', (newMail) => {
-      console.log('new mail', newMail);
+      // setTimeout(() => {
```

Comment:
remove commented code.

Reply:
I kept these consoles for the team members, so the can see what type of data are they dealing with,   in order to handle it